### ISSUE TITLE
Missing .fc-event-past and isPast is false on events ending yesterday

### ISSUE THREAD

**Issue by r20** [fullcalendar/fullcalendar#6486]
### Reduced Test Case

Bug can be seen at https://codepen.io/r20-the-styleful/pen/RwVMgey

### Bug Description

With the dayGridMonth view, an event on the day before today does not have the .fc-event-past css class.
See the codepen reproduction, on "month" view.  A css rule reduces opacity for fc-event-past and you can see "Yesterday event" does not have that class.
As a workaround, I thought of using .fc-day.past, but that won't work for cases like the event titled "Yesterday and today" because although it's within the td.fc-day-past it spans to today so it should NOT have .fc-event-past.

I noticed this in the listMonth and listWeek views as well.  The timeGridWeek view seems to work as expected.

**Comment by acerix** [comment]
It seems like a bug, the comparison for `isPast` should use `<=` since end dates are exclusive:

https://github.com/fullcalendar/fullcalendar/blob/65090506f8f6fe3101f64bb147cfa0e98253f7cd/packages/common/src/component/event-rendering.ts#L279

As a workaround you can set a class from eventClassNames:

https://codepen.io/acerix/pen/gOWzbPQ?editors=0010

**Comment by camsteffen** [comment]
Related or same issue: #6120

**Comment by fnagel** [comment]
I can confirm this bug in v5.11.4 and the workaround provided by @acerix. Any chance for a bugfix in the actual code?

If you need the internal variables to be fixed too, try this:
```
eventClassNames: function(arg) {
    const end = arg.event.end ?? arg.event.start;
    
    if (end <= todayStart) {
      arg.isPast = true;
      arg.event.isPast = true;
      return 'fc-event-past';
    }
  },    
```

**Comment by simoncodearcs** [comment]
+1 this is still an issue. 
Recently upgraded to v6.1.10 and decided to leverage the isPast boolean instead of calculating whether an event was in the past. Think I'll revert to previous calculations.

**Comment by arshaw** [comment]
This has been fixed in [v6.1.11](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.1.11)

Updated repro:
https://codepen.io/arshaw/pen/ZEPZovW

@acerix , your assessment of the bug was correct, thanks!
